dwave/cloud/client/base.py

```diff
@@ -34,6 +34,7 @@
 
 """
 
+import io
```

Comment:
Nothing to be done, but I hate how typing balloons the number of imports.

Reply:
Well, you can always go back to type comments. :slightly_smiling_face:  Or use "forward references" (quoted annotations)..